plugin/admin/src/components/Usage/index.js

```diff
@@ -0,0 +1,72 @@
+import React, { memo } from 'react'
+import {
+  Box,
+  Card,
+  CardBody,
+  CardContent,
+  CardBadge,
+  CardSubtitle,
+  CardTitle,
+  ProgressBar,
+} from '@strapi/design-system'
+import { useIntl } from 'react-intl'
+
+import useUsage from '../../Hooks/useUsage'
+import { getTrad } from '../../utils'
+
+const UsageOverview = () => {
+  const { usage } = useUsage()
+
+  const { formatMessage } = useIntl()
+
+  if (!(typeof usage?.count === 'number' && typeof usage?.limit === 'number')) {
+    return (
+      <Card marginTop={4}>
+        <CardBody>
+          <CardTitle>
+            {formatMessage({
+              id: getTrad('usage.failed-to-load'),
+              defaultMessage: 'Failed to load Usage data',
+            })}
+          </CardTitle>
+        </CardBody>
+      </Card>
+    )
+  }
+
+  return (
+    <Card marginTop={4}>
+      <CardBody style={{ alignItems: 'center' }}>
+        <CardTitle>
+          {' '}
+          {formatMessage({
+            id: getTrad('usage.title'),
+            defaultMessage: 'Usage',
+          })}
+          :
+        </CardTitle>
+        <CardContent paddingLeft={2} style={{ flexGrow: '1' }}>
+          <CardSubtitle>
+            <Box background="neutral150" padding={2}>
```

Comment:
changed it up a bit. Still had to keep the background behind the progress bar, because it would otherwise become partly invisible.

For now, I think it would be best to just drop Usage information entirely if the provider does not provide this information